articles/cloud-computing-cost-optimization-with-cloud-bursting/index.md

```diff
@@ -0,0 +1,73 @@
+Organizations across all industries have appreciated the significance of data in decision making and business development. They are now running more data workloads and analytics than ever before.
+
+But an overload of these activities may cause a system crash, workloads freeze, and ultimately downtime. This happens mainly when an organization&#39;s [cloud computing](/engineering-education/introduction-to-cloud-computing/) resources such as storage, memory, and CPU, exceed the available system capacity.
+
+Many organizations have resulted to cloud bursting to address this issue of system overload while saving on costs. With cloud bursting, organizations offload private cloud and on-premises traffic into the public cloud, thus meeting peak demand.
+
+This article will discuss the challenges and benefits of cloud bursting and show you how to optimize costs and improve cloud burst platforms' effectiveness.
+
+### What is cloud bursting?
+
+Businesses usually experience peak and off-peak seasons. They need to expand their capacity during peak seasons to handle the high demand at the time. Cloud bursting comes to handle peak demands and meet the expanding capacity requirement when private cloud resources are overwhelmed. Cloud bursting deploys applications normally running on a private cloud into the public cloud dynamically.
+
+Third-party providers offer computing services over the public internet, known as the [public cloud](https://www.citrix.com/glossary/what-is-public-cloud.html). Public cloud computing services are either free or sold on-demand and thus readily available at low costs.
+
+On the other hand, the [private cloud](https://www.citrix.com/glossary/what-is-private-cloud.html) refers to computing services offered on a private internal network or over the internet to only selected users rather than the general public. Private clouds provide benefits such as self-service, elasticity, and scalability.
+
+Usually, you would need to overbuild physical infrastructure to increase your capacity to meet fluctuating demand peaks. Luckily, cloud bursting eliminates the need for all this by ensuring that private clouds are affordable. It does this by rightsizing private clouds in terms of storage and computes to accommodate rising demands and ensuring that the public cloud handles the peaks through a pay-as-you-use subscription.
```

Comment:
The algorithm that decides the amount of compute power allocation is pretty interesting. You may want to touch upon it here.

Reply:
> compute power allocation
@lalith1403 
Which is this algorithm, please. I don't seem to understand this.

Reply:
You may ignore this for now. Btw, the Algorithms are called resource allocation algorithms.